src/AccessibilityInsights/MainWindow.xaml.cs

```diff
@@ -701,18 +703,46 @@ private void btnAccountConfig_Click(object sender, RoutedEventArgs e)
         }
 
         /// <summary>
-        /// Initialize server integration and try logging in implicitly 
-        /// to the saved connection in the configuration if it exists.
+        /// Set saved issue reporter and try restoring configuration if it exists.
         /// </summary>
-        private void ConnectToSavedServerConnection(Action callback = null)
+        private async void RestoreConfigurationAsync()
```

Comment:
Won't an Exception in the async method crash the app?

Reply:
Yes, but that's true with a lot of code in MainWindow.xaml.cs. Is there a particular part of this you're worried about? I'm thinking maybe 
`await issueReporter.RestoreConfigurationAsync(serializedConfig).ConfigureAwait(true);`
should be try-caught?

Reply:
Exceptions gets handled if the caller uses await or .ConfigureAwait or even .Result. Exceptions are not handled if you just call an async method and do nothing with it. Code like this in the AutoUpdate modules include their own try/catch block inside the async method so that the caller is free to do whatever is necessary. 

I don't have a specific line of code that concerns me, but since any exception will likely kill the process, it seems worth thinking about